engine/src/flutter/shell/platform/windows/flutter_windows_view.cc

```diff
@@ -665,9 +674,12 @@ void FlutterWindowsView::OnFramePresented() {
     case ResizeState::kFrameGenerated: {
       // A frame was generated for a pending resize.
       // Unblock the platform thread.
-      resize_status_ = ResizeState::kDone;
+      engine_->task_runner()->PostTask([this] {
```

Comment:
I understand the platform thread task wakes up the platform thread, but why does updating `resize_status_` need to be inside the platform thread task? Could we instead update `resize_status_` on the raster thread? If this is intentional, this might be worth a quick comment as it's not immediately obvious to me :)

Reply:
It's not a hard requirement, I can move it out if this seems too confusing.

Reply:
Done.